.travis-ocaml.sh

```diff
@@ -99,6 +99,8 @@ install_ocaml () {
 
 install_opam2 () {
     case $TRAVIS_OS_NAME in
+        freebsd)
+            sudo pkg install -qy ocaml-opam ;;
```

Comment:
would it be ok to find some ways of having something similar to what happens on linux or macos?

Reply:
Yes, it's pretty easy, we just need to add freebsd in the releases here : https://github.com/ocaml/opam/releases
I have no idea of how to do this…

Reply:
let's just give up on this for the moment and take the opam from the FreeBSD distribution. I forbid users from setting OPAM_VERSION or LOCAL_INSTALL. We can always allow them in the future.